schema/src/main/protobuf/data_access_event.proto

```diff
@@ -17,6 +17,12 @@ message FsEvent {
     string uri = 4;
     uint64 method_duration_millis = 5;
     string hdfs_user = 6;
+    enum Status {
```

Comment:
maybe put this at a higher level so that other events may use it

Reply:
I would avoid to share for easier evolutivity